binding-macro/src/service.rs

```diff
@@ -275,7 +293,7 @@ fn get_genesis_body(item: &ImplItemMethod) -> proc_macro2::TokenStream {
 
                 quote!{
                     let payload: #payload_ident = serde_json::from_str(&_payload)
-                                    .map_err(|e| protocol::traits::BindingMacroError::JsonParse(e))?;
+                    .unwrap_or_else(|e| panic!("service macro decode payload failed: {:?}", e));
```

Comment:
Panic if serialization fails? 

So, if I randomly pass a wrong string, the chain will crash?

Reply:
This is for `fn init_genesis` only, if init genesis payload is wrong, i think should panic and check( make it right ) genesis config and code

Reply:
right